src/native/fs/fs_napi.cpp

```diff
@@ -77,6 +77,7 @@ struct FSWorker : public Napi::AsyncWorker
     std::string _backend;
     std::string _desc;
     int _errno;
+    int thresholdInSec = 1 * 60;
```

Comment:
Probably 1 min is way too much.
@guymguym @jeniawhite What do you think the threshold should be?

Reply:
I would expect to measure milliseconds and not seconds/minutes.
The only thing is that we are measuring the whole work function (includes writing buffers, reading MD, reading directories which include multiple POSIX operations that might take a bit more time).
But still, I think the granularity should be in milliseconds.

Reply:
That's the point, syscall will take milliseconds, but the whole write?
In any case, as this is only a printing, We will set it to 100 milliseconds and if we need (get too many printings) we tweak the threshold.

Reply:
Testing upload of 25MB file:
```
2021-06-21 07:33:08.302061 [PID-9/TID-9] [L0] FS::FSWorker::Begin: FileWritev _wrap->_path=/nsfs/ns/.noobaa-nsfs_60cf4e763f991c002842dddb/uploads/f20c0854-7a49-43d2-8c31-0f2231cee936 total_len=8388608 buffers_len=512
2021-06-21 07:33:08.302230 [PID-9/TID-18] [L0] timeBefore 1624260788302
2021-06-21 07:33:08.305321 [PID-9/TID-18] [L0] timeAfter 1624260788305
2021-06-21 07:33:08.907173 [PID-9/TID-9] [L0] FS::FSWorker::Begin: FileWritev _wrap->_path=/nsfs/ns/.noobaa-nsfs_60cf4e763f991c002842dddb/uploads/f20c0854-7a49-43d2-8c31-0f2231cee936 total_len=8388608 buffers_len=510
2021-06-21 07:33:08.907354 [PID-9/TID-17] [L0] timeBefore 1624260788907
2021-06-21 07:33:09.399878 [PID-9/TID-17] [L0] timeAfter 1624260789399
2021-06-21 07:33:09.399975 [PID-9/TID-17] [L0] FS::FSWorker::Execute: FileWritev _wrap->_path=/nsfs/ns/.noobaa-nsfs_60cf4e763f991c002842dddb/uploads/f20c0854-7a49-43d2-8c31-0f2231cee936 total_len=8388608 buffers_len=510  took: 492 milliseconds
2021-06-21 07:33:09.799042 [PID-9/TID-9] [L0] FS::FSWorker::Begin: FileWritev _wrap->_path=/nsfs/ns/.noobaa-nsfs_60cf4e763f991c002842dddb/uploads/f20c0854-7a49-43d2-8c31-0f2231cee936 total_len=820261 buffers_len=51
2021-06-21 07:33:09.799737 [PID-9/TID-16] [L0] timeBefore 1624260789799
2021-06-21 07:33:09.899385 [PID-9/TID-16] [L0] timeAfter 1624260789899
```

As `FileWritev` can take 492 milliseconds we will set the threshold to 100 milliseconds